sources/api/host-containers/src/main.rs

```diff
@@ -434,8 +434,21 @@ where
     Ok(())
 }
 
+fn verify_change_for_container(changed_settings: &String, container_name: &str) -> bool {
+    let settings: Vec<&str> = changed_settings.split_whitespace().collect();
+    for setting in settings.iter() {
+        // if its a host-container setting, check for specific host-container
+        // if its some other setting, return true for all host-containers, example: network
+        if setting.contains(container_name) == true || ! setting.contains("host-containers") {
```

Comment:
I just realized may be we should check for `"host-containers.<container_name>"` instead of just `container_name` to be safer, since it guarantees that we are only looking for host-containers with `container_name` name and  not any other substring containing `container_name`

Reply:
Yeah, cool. that would make it clearer. Thanks!